### DISCUSSION TITLE
Question  about Property Setter

### DISCUSSION THREAD

**Discussion by violet701** [canton7/Stylet#294] [Q&A] [answered]
```csharp
private int test = 0;
public int Test
{
    get => test;
    set
    {
        test = value;
        DoSomething();
    }
}
```
When i set the property "Test", **if the value is equals to "test",** the setter will not execute，so the function DoSomething() also not executes. What is the reason for it, and how to solve the problem whenever the value I set equals or not  to the ‘’test‘’? Thanks.

**Comment by violet701** [comment, marked as answer]
I am sorry, i do not know how to delete this conversation. And I have just found the reason.  It's Fody, it is not properly configured.

**Reply by canton7** [reply]
Good to hear, thanks for letting me know!
